files/es/learn/common_questions/set_up_a_local_testing_server/index.html

```diff
@@ -36,70 +36,87 @@ <h2 id="Archivos_locales_versus_archivos_remotos">Archivos locales versus archiv
 
 <ul>
- <li><strong>Cuentan con peticiones asincrónicas</strong>. Algunos navegadores (incluido chrome) no ejecutarán solicitudes asíncronas (consulte <a href="/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Fetching_data">Obtención de datos desde el servidor</a>) si acaba de ejecutar el ejemplo desde un archivo local. Esto se debe a las restricciones de seguridad (para obtener más información sobre la seguridad web, lee <a href="/en-US/docs/Learn/Server-side/First_steps/Website_security">La seguridad del sitio web</a>).</li>
- <li><strong>Cuenta con un lenguaje de servidor</strong>. Los lenguajes de servidor (como PHP o Python) requieren de un servidor especial para interpretar el código y entregar los resultados.</li>
+ <li><strong>Cuentan con peticiones asincrónicas</strong>. Algunos navegadores (incluido chrome) no ejecutarán solicitudes asíncronas (consulte <a href="/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Fetching_data">Obtención de datos desde el servidor</a>) si se ejecuta el ejemplo desde un archivo local. Esto se debe a restricciones de seguridad (para obtener más información sobre la seguridad web, lee <a href="/es/docs/Learn/Server-side/First_steps/Website_security">La seguridad del sitio web</a>).</li>
+ <li><strong>Cuentan con un lenguaje de servidor</strong>. Los lenguajes de servidor (como PHP o Python) requieren de un servidor especial para interpretar el código y entregar los resultados.</li>
+ <li><strong>Incluyen otros archivos</strong>. Los navegadores comunmente tratan a los pedidos para cargar recursos formulados utilizando el esquema <code>file://</code>, como solicitudes de recursos de origenes cruzados (<em>cross-origin requests</em>). Por ello, si se carga un archivo local que incluye otros archivos locales, puede dispararse un error <a href="/en-US/docs/Glossary/CORS">CORS</a>.</li>
 </ul>
 
 <h2 id="Ejecutando_un_servidor_HTTP_local_simple">Ejecutando un servidor HTTP local simple</h2>
 
-<p>Para evitar el problema de las solicitudes asíncronas, necesitamos probar estos ejemplos ejecutándolos a través de un servidor web local. Una de las maneras más fáciles de hacer esto para nuestros propósitos es usar el módulo <code>SimpleHTTPServer</code> de Python.</p>
+<p>Para evitar el problema de las solicitudes asíncronas, necesitamos probar estos ejemplos ejecutándolos a través de un servidor web local. Una de las maneras más fáciles de hacer esto para nuestros propósitos es usar el módulo <code>http.server</code> de Python.</p>
+
+<div class="note">
+  <p><strong>Nota</strong>: Versiones más antiguas de Python (hasta la version 2.7) proveían un módulo similar llamado <code>SimpleHTTPServer</code>. Si estás usando Python 2.x, puedes seguir esta guía reemplazando todos los usos de <code>http.server</code> por <code>SimpleHTTPServer</code>. Sin embargo, recomendamos que uses la última versión de Python.</p>
+  </div>
```

Comment:
Creo que este espacio puede estar demás.
```suggestion
</div>
```